modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/js/ManagementToolbarPropsTransformer.js

```diff
@@ -0,0 +1,252 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import {
+	addParams,
+	navigate,
+	openModal,
+	openSelectionModal,
+} from 'frontend-js-web';
+
+export default function propsTransformer({
+	additionalProps: {
+		downloadEntryURL,
+		editEntryURL,
+		folderConfiguration,
+		openViewMoreFileEntryTypesURL,
+		selectFileEntryTypeURL,
+		selectFolderURL,
+		trashEnabled,
+		viewFileEntryTypeURL,
+	},
+	portletNamespace,
+	...otherProps
+}) {
+	const processAction = (action, url) => {
+		if (!action) {
+			return;
+		}
+
+		const form = document.getElementById(`${portletNamespace}fm2`);
+
+		if (!form) {
+			return;
+		}
+
+		form.setAttribute('method', 'post');
+
+		const actionInputElement = form.querySelector(
+			`#${portletNamespace}javax-portlet-action`
+		);
+
+		if (actionInputElement) {
+			actionInputElement.setAttribute('value', action);
+		}
+
+		const commandInputElement = form.querySelector(
+			`#${portletNamespace}cmd`
+		);
+
+		if (commandInputElement) {
+			commandInputElement.setAttribute('value', action);
+		}
+
+		submitForm(form, url, false);
+	};
+
+	const checkin = () => {
```

Comment:
Perhaps capitalize the `I`. `checkin` => `checkIn`?

Reply:
Why not, does this have any advantage?

Reply:
`checkIn` is grammatically correct, it's the same as `signin` instead of `signIn`, which one would you use if this was "Sign in" instead of "Check in"

Reply:
Ok, since I need to touch this PR anyway, I will make this change